accounts/abi/bind/precompilebind/precompile_config_template.go

```diff
@@ -48,6 +54,14 @@ func NewConfig(blockTimestamp *uint64{{if .Contract.AllowList}}, admins []common
 	}
 }
 
+func (c * Config) MarshalBinary() ([]byte, error) {
```

Comment:
can we add some comments here why these are important, how these can be implemented etc?